### DISCUSSION TITLE
Get all users with multiple role.

### DISCUSSION THREAD

**Discussion by Dasinfomedia2** [spatie/laravel-permission#1708] [Q&A] [answered]
below code gets all users for single role only.
`$customers = User::role('customer');   `

I want to do something like below.
`$customers = User::role(['customer','Operator','Technician']);`

How do I get with multiple roles? is there any feature like this?

**Comment by drbyte** [comment, marked as answer]
Yes, that is built-in already, exactly as you've quoted, except you've forgotten to call `->get()` on the end of your query.

**Reply by Dasinfomedia2** [reply]
Thanks for the reply. So it takes an array as well. Great.

Just for the example, I didn't notice `->get()` is missing in copy-pasting. :)
